nmostesting/NMOSTesting.py

```diff
@@ -1115,8 +1115,8 @@ def main(args):
         repo = git.Repo(".")
         TOOL_VERSION = repo.git.rev_parse(repo.head.object.hexsha, short=7)
     except git.exc.InvalidGitRepositoryError:
-        TOOL_VERSION = "Unknown"
-
+        TOOL_VERSION = os.getenv('TOOL_VERSION', 'Unknown')
+        

```

Comment:
Just deleting whitespace from blank line...
```suggestion
        TOOL_VERSION = os.getenv('TOOL_VERSION', 'Unknown')

```